app/services/twilio/incoming_message_service.rb

```diff
@@ -46,6 +46,10 @@ def formatted_phone_number
     TelephoneNumber.parse(phone_number).international_number
   end
 
+  def message_body
+    params[:Body].delete("\u0000")
```

Comment:
@jordan-brough This will throw an error if the body doesn't exist in the param. This [PR](https://github.com/chatwoot/chatwoot/pull/7910) will fix the issue.